WordPress/Classes/ViewRelated/Post/PostTagPickerViewController.swift

```diff
@@ -0,0 +1,402 @@
+import Foundation
+import WordPressShared
+
+class PostTagPickerViewController: UIViewController {
+    private let originalTags: [String]
+    var onValueChanged: ((String) -> Void)?
+    let blog: Blog
+    let keyboardObserver = TableViewKeyboardObserver()
+
+    init(tags: String, blog: Blog) {
+        originalTags = PostTagPickerViewController.extractTags(from: tags)
+
+        self.blog = blog
+        super.init(nibName: nil, bundle: nil)
+        textView.text = normalizeInitialTags(tags: originalTags)
+        textViewDidChange(textView)
+        title = NSLocalizedString("Tags", comment: "Title for the tag selector view")
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    fileprivate let textView = UITextView()
+    private let textViewContainer = UIView()
+    private let shadow = ShadowView()
```

Comment:
Is there any reason to use our own custom class instead of doing something like:

```
textViewContainer.layer.masksToBounds = false
textViewContainer.layer.shadowOffset = CGSize(width: 0, height: 2)
textViewContainer.layer.shadowRadius = 2
textViewContainer.layer.shadowOpacity = 0.5
textViewContainer.layer.shadowColor = WPStyleGuide.greyDarken30().cgColor
```
And add the tableview as a subview before the textViewContainer? I think the end result is quite good and we avoid adding an extra view to the hierarchy (with all the constraints and that I think that's a big plus)

![simulator screen shot may 11 2017 10 18 04](https://cloud.githubusercontent.com/assets/5558824/25951176/4c17011c-3633-11e7-8334-acebcbf28d6b.png)